inc/settings.php

```diff
@@ -226,6 +226,10 @@ function vantage_theme_settings(){
 		'description' => __('Show an author box below each blog post.', 'vantage')
 	) );
 
+	$settings->add_field('blog', 'comment_author', 'text', __('Post Author\'s Comments', 'vantage'), array(
+		'description' => __('Text displayed as a label next to the post author\'s comments.', 'vantage')
```

Comment:
It's best practice to keep slashes out of the translation string. So something like this would be better:

`__("Text displayed as a label next to the post author's comments.", 'vantage')`